week2/materials/homework/assignments.md

```diff
@@ -4,38 +4,38 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/g3-week2-tithi?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `Though I see, the website already supports https. But I found one article(https://sectigostore.com/page/steps-to-install-an-ssl-certificate-on-aws-ec2-instance/), where I can follow the 5 steps to install SSL certificate to achieve this.`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `1370 usd, as first 50tb costs $.023 and next 1tb costs $.022`
 
-Total size of website: `?`
+Total size of website: `498.092 KB`
 
 **Assignment 7:**
 
 Write down the main cost factors for S3
 
-brief description: `?`
+brief description: `Main cost factors are: storage pricing, request and data retrieval pricing, data transfer and transfer acceleration pricing, data management and analytics pricing, and the price to process your data with S3 Object Lambda`
 
 **Assignment 8:**
 
 The file `week2/assignments/products.json` needs to be uploaded Inside the bucket `hyf-serverless-course-week-2`. Upload the file through the CLI and write down the command needed.
 
-docs: 
+docs: `aws s3 cp materials\homework\products.json s3://g3-week2-tithi`
```

Comment:
I think you linked to a command, and not the actual documentation of the cp command.